File: JanperCab/3-Application/Dtos/DuraformComponent/DuraformComponentPriceDto.cs
namespace _3_Application.Dtos.DuraformComponent
{
    public class DuraformComponentPriceDto
    {
        public int Id { get; set; }

        public decimal UnitPrice { get; set; }

        public decimal SubTotal { get; set; }

        public decimal TotalDiscount { get; set; }

        public decimal TotalPrice { get; set; }
    }
}

File: JanperCab/5-JanperCab/ClientApp/src/app/_models/duraform-component/DuraformComponentPriceDto.ts
import { DuraformComponentDto } from 'src/app/_models/duraform-component/DuraformComponentDto';
export class DuraformComponentPriceDto {
  id: number;

  unitPrice: number;
  subTotal: number;
  totalDiscount: number;
  totalPrice: number;

  constructor(duraformComponent: DuraformComponentDto) {
    this.id = duraformComponent.id;
    this.unitPrice = duraformComponent.unitPrice;
    this.subTotal = duraformComponent.subTotal;
    this.totalDiscount = duraformComponent.totalDiscount;
    this.totalPrice = duraformComponent.totalPrice;
  }
}
